.github/workflows/release.yaml

```diff
@@ -0,0 +1,74 @@
+---
+name: Release
+
+run-name: release ${{ inputs.tag }}
+
+on:
+  workflow_dispatch:
+    inputs:
+      tag:
+        description: 'Tag to be released'
+        required: true
+
+permissions:
+  packages: write
+  id-token: write
+  contents: read
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+      - name: Ensure the branch is protected
+        run: |
+          if [[ "${{ github.ref_protected }}" == "false" ]]; then
```

Comment:
can we be consistent on bash script for the use of `[[` vs `[`

Reply:
yes